### DISCUSSION TITLE
Translations for Person Card

### DISCUSSION THREAD

**Discussion by thomaswidmer** [microsoft-search/pnp-modern-search#1714] [Q&A] [answered]
Is it possible to integrate localization of person cards (called by the Search Results Webpart)? Or is a translation already integrated in these cards (e.g. German translation for titles inside the card)? Actually, we only see English expressions in the person card.

Further, is it possible to call Windows standard App for Chat instead of the actual static link to Microsoft Teams?

Thanks for any suggestions/feedbacks.

**Comment by wobba** [comment, marked as answer]
The Graph Toolkit does not have translations in `mgt-person`, if that is what you are using. You should file a localization request with them at https://github.com/microsoftgraph/microsoft-graph-toolkit/issues

**Reply by thomaswidmer** [reply]
Thank you for the immediate response. Interesting. That means, that the following feature is not yet integrated in the Graph Toolkit: https://mgt.dev/?path=/docs/components-mgt-person-card--localization

However, I will open open request as you said. Thanks again.

**Reply by wobba** [reply]
Thank you for pointing to that link. It means that if someone want to add translations we can load them up using
`import { LocalizationHelper } from '@microsoft/mgt';` as mentioned at https://docs.microsoft.com/en-us/graph/toolkit/customize-components/localization

**Comment by stevebeauge** [comment]
Is there, three years later, an actual solution to this issue? 
Unless I missed something, there's still no way to pass translations to the mgt toolkit when using pnp search.
thanks

**Reply by kasperbolarsen** [reply]
The MGT components are just components we are using in the PnP Modern Search project,  and we would love to see a localizable version,  but it is not a part of this project to create that component

**Reply by patrikhellgren** [reply]
I think that because we use mgt-person and by that also mgt-person-card and other sub-components in the out of the box layouts maybe we should add the translations for that in the webpart since it in fact a part of the solution. Just did this for another project so should be a quickie and I could submit a PR for that.
Your thoughts on that @kasperbolarsen?

**Reply by kasperbolarsen** [reply]
I guess that would mean that we pick up a dependency as MGT might be changed without we might notice it?
Yet, if the only consequence would be that a new label wouldn't be translated,  then I am all for it.

**Reply by patrikhellgren** [reply]
I just submitted a PR that addresses this (#4313)
The version dependency of MGT in PnP Modern Search cannot be updated without a PR and if there were some new localizable strings or if they changed names those would just appear in english until someone added translations for them.

**Reply by stevebeauge** [reply]
Awesome !

**Comment by wobba** [comment]
Solved in #4313
